src/utils/handleError.ts

```diff
@@ -0,0 +1,9 @@
+export const handleErrorWithTimeout = (
+  errorMessage: string,
+  setErrorMessage: React.Dispatch<React.SetStateAction<string>>,
+) => {
+  setErrorMessage(errorMessage);
+  setTimeout(() => {
+    setErrorMessage('');
+  }, 3000);
+};
```

Comment:
you need to clear timeout and handle multiple timeouts. [here you can read more about it](https://www.freecodecamp.org/news/how-to-use-settimeout-in-react-using-hooks/)